sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -25,6 +25,7 @@ This table shows the relationship between SDK versions and supported API version
 
 |SDK version|Supported API version of service
 |-|- |
+|5.2.0-beta.1 | 3.0, 3.1, 3.2-preview.1 (default)
```

Comment:
I can't comment on it, but line 22 needs to be updated too

Reply:
![image](https://user-images.githubusercontent.com/7576220/128087171-ca13af3b-4d4c-413a-8e06-c3773858e1a6.png)

I thought the comment was referring to the package in the installation instructions above:

> dotnet add package Azure.AI.TextAnalytics

From my understanding, it will add the latest GA package. That's why I left it as `v3.1`. Does it make sense? Is it better to make it clear that in order to install the preview version we need to specify it?

Reply:
Ohhh I see how that can be confusing. What do you think of just removing that line? and then with the table, the user can see what SDK they have and do the mapping

Reply:
Sounds good! Updated.